packages/tools/pluggable-widgets-tools/scripts/validate-version.js

```diff
@@ -0,0 +1,21 @@
+const parser = require("fast-xml-parser");
```

Comment:
Try to use the libs we already have in the PIW-Tools. We are doing the same parse in the typing generator where we read Widget.xml and convert properties to Typings.

https://github.com/mendix/widgets-resources/blob/314f1dfbd8b83f7653bdf6331dae67dbefaad76a/packages/tools/pluggable-widgets-tools/src/typings-generator/index.ts#L36

Reply:
Thanks, I didn't see this is dependencies! I will change 😃

Reply:
Updated.

Reply:
Looks good. I see you used:
```const { parseStringPromise } = require("xml2js");```

Could you do the same for the typing generator, since you are changing the piw tools already?

Reply:
@IIsaku done 👍